src/scripts/components/VideoOverlay.js

```diff
@@ -40,67 +41,69 @@ type Props = {
   activePlugin: ?PlayerPlugin
 }
 
+const Z_INDEX_OVERLAY: string = '1'
+const Z_INDEX_CONTROLS: string = '2'
+const Z_INDEX_OVERLAY_SHADOW: string = '1'
+const Z_INDEX_CENTRALIZEDCONTENT: string = '2'
+const Z_INDEX_TITLE: string = '3'
+const Z_INDEX_BUTTONS: string = '4'
+
+const UpDownElements = css`
```

Comment:
What is `UpDownElements`? Wondering if there is a more descriptive name we might be able to use